### ISSUE TITLE
Unpinning Icon not working immediately in IconTaskList

### ISSUE THREAD

**Issue by yursan9** [solus-project/budgie-desktop#1663]
**Operating System**: e.g. Solus 3.9999
**Budgie version**: i.e. Budgie 10.4 e1922ed6

**The issue encountered**

When unpinning icon from IconTaskList, it's not working immediately. I need to open and close the application to hide it.

**Steps to reproduce the issue**

![unpin_icon_not_working](https://user-images.githubusercontent.com/17114559/49782547-4660f780-fd49-11e8-8648-c7ce2e29495b.gif)

**Comment by kyrios123** [comment]
Funny I wantedo to log the exact same issue on the same application.
Actually you need to open it and it will disappear.

Also, how about a pin icon instead of a star ?

**Comment by JoshStrobl** [comment]
> Also, how about a pin icon instead of a star ?

No. See [this post](https://www.reddit.com/r/SolusProject/comments/a39rpr/budgie_105_icontasklist_popover_preview/eb8tauw/) and keep things on topic.
